cranelift/isle/isle/src/sema.rs

```diff
@@ -1687,12 +1701,13 @@ impl TermEnv {
                         &mut bindings,
                         /* is_root = */ true,
                     ));
-                    let iflets = unwrap_or_continue!(self.translate_iflets(
-                        tyenv,
-                        rule_term,
-                        &rule.iflets[..],
-                        &mut bindings,
-                    ));
+                    let iflets = rule
+                        .iflets
+                        .iter()
+                        .filter_map(|iflet| {
+                            self.translate_iflet(tyenv, rule_term, iflet, &mut bindings)
+                        })
+                        .collect();
```

Comment:
It was a little surprising that this change was equivalent until I noticed that `translate_iflet` unconditionally returned a `Some` result :)